src/web/docusaurus/docusaurus.config.js

```diff
@@ -120,7 +120,7 @@ const config = async () => {
               items: [
                 {
                   label: 'Documentation',
-                  href: 'https://docusaurus.io/docs/',
+                  href: '/docs/overview',
```

Comment:
```suggestion
                  to: '/docs/overview',
```

Reply:
Based on doc `to` is preferable for internal navigation

Reply:
Yes, thank you @Kevan-Y .